src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -15,6 +15,11 @@
     public static final String MESSAGE_INVALID_PROPERTY_DISPLAYED_INDEX = "The property index provided is invalid";
     public static final String MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX =
             "The appointment index provided is invalid";
+    public static final String MESSAGE_MISSING_PROPERTY_APPOINTMENT =
+            "To %s an appointment, use %s appointment. To %s a property, use %s property";
+    public static final String MESSAGE_MISSING_ALL_PROPERTY_APPOINTMENT =
+            "To %s all appointments and properties, use %s all. "
+                    + "To %s all appointments, use %s appointment. To %s all properties, use %s property";
```

Comment:
Food for thought: how about finding client and the other extra stuff (finding by postalcode, etc) yuchen is adding? Or do we only prompt appointment and property because they are more important?

Reply:
maybe we can tackle find in another pr?

Reply:
ok